amplitude/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+**What does this PR do?**
```

Comment:
Same here, the PR applies to all destinations.

Reply:
Definitely. On it